src/components/MerchantsPage/DescriptionBox.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import styles from './styles.module.scss';
+import merchantIcon from './images/merchantIcon.png';
+
+const Container = styled.div`
+  text-align: center;
+`;
+
+const IMG = styled.img`
+  width: 45%;
+`;
+
+const UL = styled.ul`
+  padding: 0 5px 0 20px;
+`;
+
+const BP = styled.li`
+  text-align: left;
+  padding: 5px 0;
+`;
+
```

Comment:
perhaps better name conventions for the tags for readability? I also prefer having the styles below the components just because if i ever want to look at a component, more often than not i'm going to be looking at it to change the react code itself instead of its styles.